### DISCUSSION TITLE
SkyWalking alarm module send dingding message，how to exclude url？

### DISCUSSION THREAD

**Discussion by andotorg** [apache/skywalking#9510] [Q&A]
SkyWalking alarm module send dingding message，
How to exclude some URLs from meeting the rules without sending alarm messages

**Comment by pg-yang** [comment]
[Exclude names](https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-alarm/#entity-name )

**Reply by andotorg** [reply]
please write example
api/sse/getInfo
api/sse/getName
Exclude URLs with  `sse`

**Reply by pg-yang** [reply]
You should use `Exclude names regex` , And try it yourself .  
reference  [RunningRule.java](https://github.com/apache/skywalking/blob/master/oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java#L179)

**Reply by andotorg** [reply]
rules:
    endpoint_relation_resp_time_rule:
    metrics-name: endpoint_relation_resp_time
    threshold: 1
    op: ">"
    period: 1
    count: 1
    message: 端点：{name}
    excludeNamesRegex: "^(?=.*sse).*$"

this not effect

**Reply by pg-yang** [reply]
The key is `exclude-names-regex`  , and you could use `validate` validate your regex .

**Reply by andotorg** [reply]
Yes，have effect, thanks
